doc/howto/storage_volumes.md

```diff
@@ -6,6 +6,37 @@
 
 See the following sections for instructions on how to create, configure, view and resize {ref}`storage-volumes`.
 
+## View storage volumes
+
+You can display a list of all available storage volumes and check their configuration.
+
+To list all available storage volumes, use the following command:
+
+    lxc storage volume list
+
+To display the storage volumes for all projects (not only the default project), add the `--all-projects` flag.
+
+You can also display the storage volumes in a specific storage pool:
+
+    lxc storage volume list my-pool
+
+The resulting table contains, among other information, the {ref}`storage volume type <storage-volume-types>` and the {ref}`content type <storage-content-types>` for each storage volume.
+
+```{note}
+Custom storage volumes might use the same name as instance volumes (for example, you might have a container named `c1` with a container storage volume named `c1` and a custom storage volume named `c1`).
```

Comment:
```suggestion
Custom storage volumes can use the same name as instance volumes. For example, you might have a container named `c1` with a container storage volume named `c1` and a custom storage volume named `c1`.
```

Reply:
Let's try to keep parenthesized text concise, for readability's sake. If what's parenthesized is longer than the sentence it's in, it likely deserves to be freed from its prison.